prepare/cards/sst2.py

```diff
@@ -49,5 +49,5 @@
     templates="templates.one_sent_classification",
 )
 
-test_card(card)
+test_card(card, debug=False)
```

Comment:
again, just checking, if that's the default, should it be removed?

Reply:
Left as an example.